# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1498347779 14400
#      Sat Jun 24 19:42:59 2017 -0400
# Node ID 2fa6013799fa3b6a3fc3a37cb9d9e69857a8676a
# Parent  8bba52bd04d2e40437fb69ae9756241eb0b3c4f8
* gzip.cc: Tag %!xtest with bug number.

diff --git a/libinterp/dldfcn/gzip.cc b/libinterp/dldfcn/gzip.cc
--- a/libinterp/dldfcn/gzip.cc
+++ b/libinterp/dldfcn/gzip.cc
@@ -766,17 +766,17 @@ The optional output @var{filelist} is a 
 %!
 %!  md5_z = hash ("md5", fileread (z_file));
 %!  unlink_or_error (z_file);
 %!  uz_z_filelist = z.unzip (z_z_file);
 %!  assert (uz_z_filelist, {z_file}) # bug #48598
 %!  assert (exist (z_z_file), 2) # bug #48597
 %!  assert (hash ("md5", fileread (z_file)), md5_z)
 %!endfunction
-%!xtest <48597> run_test_function (@test_z_z)
+%!test <48597> run_test_function (@test_z_z)
 
 %!function test_xzip_dir (test_dir, z) # bug #43431
 %!  fpaths = fullfile (test_dir, {"test1", "test2", "test3"});
 %!  md5s = cell (1, 3);
 %!  for idx = 1:numel(fpaths)
 %!    create_file (fpaths{idx}, rand (100, 1));
 %!    md5s(idx) = hash ("md5", fileread (fpaths{idx}));
 %!  endfor
@@ -800,17 +800,17 @@ The optional output @var{filelist} is a 
 %!      uz_filelist(idx) = z.unzip (z_filelist{idx});
 %!    endfor
 %!  endif
 %!  assert (sort (uz_filelist), fpaths(:)) # bug #48598
 %!  for idx = 1:numel(fpaths)
 %!    assert (hash ("md5", fileread (fpaths{idx})), md5s{idx})
 %!  endfor
 %!endfunction
-%!xtest run_test_function (@test_xzip_dir)
+%!test <48598> run_test_function (@test_xzip_dir)
 
 %!function test_save_to_dir (test_dir, z)
 %!  filename = "test-file";
 %!  filepath = fullfile (test_dir, filename);
 %!  create_file (filepath, rand (100, 1));
 %!  md5 = hash ("md5", fileread (filepath));
 %!
 %!  ## test with existing and non-existing directory
